dist/keen-tracking.min.js

```diff
@@ -6,4 +6,4 @@
-//# sourceMappingURL=keen-tracking.min.js.map
\ No newline at end of file
+//# sourceMappingURL=/Users/arturpryka/SourceFiles/keen-tracking.js/dist/keen-tracking.min.js.map
```

Comment:
this is leaking the path in which the library was developed for the last user running the build. I don't think that's a good idea for released artifacts.